elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -105,6 +107,11 @@ class PfeAccordion extends PFElement {
 
       this._observer.observe(this, { childList: true });
     });
+    if ((!this.hasAttribute("pfe-disclosure")) && ( this._allHeaders().length === 1 )) {
+      this.setAttribute("pfe-disclosure", "true");
+      this._firstHeader().setAttribute("pfe-disclosure", "true");
```

Comment:
Is it necessary to apply this to the first header and first panel only since we do a check around it to ensure there are only one of each anyway?  Maybe just apply the attribute to all the headers and panels since there should only be one by the time we've gotten in here?

Reply:
Good point. Set a `forEach` loop through all panels and headers and applied the `pfe-disclosure="true"` tag.